Cesium.IntegrationTests/function_storage.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+
+static void worker();
+
+static void worker()
+{
+    printf("inside worker");
+}
+
+static void storage_declared_worker();
+
+void storage_declared_worker()
+{
+    printf("inside storage_declared_worker");
+}
+
+/*
+void storage_defined_worker();
+
+static void storage_defined_worker()
+{
+    printf("inside storage_defined_worker");
+}*/
```

Comment:
Why is this commented-out? Doesn't it work?

Reply:
On MSVC it’s fine. In Cesium too.

Reply:
I suspect it breaks the standard then, let's git rid of this particular test?

Perhaps there's no sense in declaring a function with no `static` linkage, and then marking an implementation as `static`.